core/ide/src/main/java/org/alice/stageide/sceneeditor/views/SceneObjectPropertyManagerPanel.java

```diff
@@ -213,94 +213,94 @@ private void addPropertyToPanel(LabelValueControllerPair propertyPair, GridBagPa
       boolean isVisible = (setter == null) || (setter.getVisibility() == null) || (setter.getVisibility() == Visibility.PRIME_TIME);
       if ((setter != null) && isVisible) {
         if (setter.getName().equalsIgnoreCase("setOpacity")) {
```

Comment:
on one hand, this file is now overrun with imps, on the other hand, this function is already really hard to read so i don't know if i care